### DISCUSSION TITLE
audioprocess and timeupdate events keep running which hang the browser

### DISCUSSION THREAD

**Discussion by sumeena** [katspaugh/wavesurfer.js#3029] [Q&A]
## Environment
 - Browser: Chrome

## Expected result
It should clear the events on its own, when stopRecording() or wavesurfer.stop() event called.

## Obtained result
It hang the browser, even replaying the audio after recording freeze the wavesurfer.

**Comment by katspaugh** [comment]
I’ve been unable to confirm if there’s a problem in wavesurfer code. These events are only emitted when audio is playing for me. Maybe you could share a minimal piece of code to replicate this?

**Comment by MahmoudKSultan** [comment]
Did you figure it out? I am also facing the same problem
